src/apps/backend/modules/config/internals/config.py

```diff
@@ -0,0 +1,156 @@
+import os
+from pathlib import Path
+from typing import Any, Optional
+import yaml
+
+
+class Config:
+    _config_dict: dict[str, Any]
+    _config_path: Path 
+
+    @staticmethod
+    def load_config() -> None:
+        Config._intialize_config()
+        Config._process_custom_environment_variables()
+        print(yaml.dump(Config._config_dict))
+        
+    @staticmethod
+    def get(key: str, default: Optional[Any] = None) -> Any:
+        keys = key.split(".")
```

Comment:
Avoid using magic string or number. You can make this code readable by storing `.` in a constant variable, something of the form - `CONFIG_KEY_SEPARATOR` or something to that effect.